src/sentry/static/sentry/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -237,6 +239,17 @@ class AbstractIntegrationDetailedView<
     return getCategories(this.featureData);
   }
 
+  renderRequestIntegrationButton(provider) {
```

Comment:
@mgaeta We don't need an input to this function. You can get name from `integrationName` and slug from `integrationSlug`.

Reply:
It's so clean now 👍